### PR TITLE
add details of the endpoint uri for "int.nugettest.org"

### PR DESCRIPTION
**Pull request by rrelyea**
found it hard to use nuget push against int w/o the endpoint uri.

### CODE REVIEW

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[de24db3](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1313/commits/de24db398952379456e7c74c87ca40a5a4c0a61f)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[de24db3](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1313/commits/de24db398952379456e7c74c87ca40a5a4c0a61f)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[de24db3](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1313/commits/de24db398952379456e7c74c87ca40a5a4c0a61f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/faqs/NuGet-FAQ.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-int.nugettest-endpoint/docs/faqs/NuGet-FAQ.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/faqs/nuget-faq?branch=pr-en-us-1313) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902132350592365-1313%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hj3EmgmiNil4rkQQQrg43k%2FNh1p37cgusYwlIiMWyF8%3D&st=2019-02-13T23%3A47%3A00Z&se=2019-03-16T23%3A52%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/faqs/NuGet-FAQ.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ NuGet recommends all packages to be signed. A design principle of package signin
 
 Note that dependencies listed for your package are generated automatically from the package itself and cannot be edited.
 
-In addition, uploading packages to [int.nugettest.org](https://int.nugettest.org) is a great way to test and validate your package without making a package available in the public gallery.
+In addition, uploading packages to [int.nugettest.org](https://int.nugettest.org) is a great way to test and validate your package without making a package available in the public gallery. (EndPoint Uri is https://apiint.nugettest.org/v3/index.json)
```
Comment (loic-sharma) [inline docs/faqs/NuGet-FAQ.md]: Instead of EndPoint, how about "Source"? That's the term we use on nuget.exe, dotnet.exe, and Visual Studio.

Comment (rrelyea) [reply docs/faqs/NuGet-FAQ.md]: The section we edited recently uses api endpoint.

https://docs.microsoft.com/en-us/nuget/faqs/nuget-faq#what-is-the-api-endpoint-for-nugetorg

following that...perhaps i should change EndPoint Uri to "API Endpoint"

Comment (joelverhagen) [reply docs/faqs/NuGet-FAQ.md]: This doesn't need to be in parenthesis.

Also, "The API endpoint...". The P is not capitalized in endpoint.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[6db1eb6](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1313/commits/6db1eb61c1e8e7dd8efe2fb9e5ae66a8d40d2eba)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[6db1eb6](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1313/commits/6db1eb61c1e8e7dd8efe2fb9e5ae66a8d40d2eba)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[6db1eb6](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1313/commits/6db1eb61c1e8e7dd8efe2fb9e5ae66a8d40d2eba)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/faqs/NuGet-FAQ.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-int.nugettest-endpoint/docs/faqs/NuGet-FAQ.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/faqs/nuget-faq?branch=pr-en-us-1313) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C14%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902140127534277-1313%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9T7EMpQtSxk4AKsixelm4VjgRrzJ9FN6Wsk1SIgElUw%3D&st=2019-02-14T01%3A23%3A58Z&se=2019-03-17T01%3A28%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**